### PR TITLE
Cross repo tests for #19561 (remove to_a from all_ems_in_zone)

### PR DESCRIPTION
**Pull request by djberg96**
Cross repo tests for https://github.com/ManageIQ/manageiq/pull/19561

Azure is the main check since it redefines the method, I added a few others for good measure.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,12 @@ matrix:
   fast_finish: true
 env:
   global:
-  - REPOS=
+  - REPOS=manageiq@19561
```
Comment (agrare) [inline .travis.yml]: @djberg96 unless `19561` is the name of a ref and not a PR you probably want `#19561`

**Comment by djberg96** [comment]
@agrare or @Fryguy I'm not following the error here.

**Comment by agrare** [comment]
@djberg96 it's because you're trying to test and override core with two different revisions.

If you want to override core with a PR you have to test with the same PR

**Comment by agrare** [comment]
`manageiq` is shorthand for `ManageIQ/manageiq@master`

**Comment by djberg96** [comment]
Whoops, fixed, thanks.

**Comment by agrare** [comment]
@djberg96 you can still test core, you just need to pass the same PR to TEST_REPO as you have in REPOS

**Comment by djberg96** [comment]
Eh, the core PR will do its own testing though, so it felt redundant.

**Comment by djberg96** [comment]
PR was merged, closing.

**Pull request closed without merging**